87.md

```diff
@@ -0,0 +1,129 @@
+NIP-87
+======
+
+Ecash Mint Discoverability
+--------------------------------
+
+`draft` `optional`
+
+This NIP describes `kind:38173` and `kind:18173`: a way to discover ecash mints, their capabilities, and people who recommend them.
+
+## Rationale
+
+Nostr's discoverability and transparent event interaction is one of its most interesting/novel mechanics.
+This NIP provides a simple way for users to discover ecash mints recommended by other users and to interact with them.
+
+### Parties involved
+
+There are three actors to this workflow:
+
+* An ecash mint operator, announces their mint and its capabilities.
+    * Publishes `kind:38173`, detailing how to connect to it and its capabilities.
+* user A, who recommends an app that handles a specific event kind
```

Comment:
You have quite a few places that still uses the word `app` or `application` when you should be using `mint`